openid-federation-1_0.xml

```diff
@@ -7719,6 +7721,169 @@ HTTP/1.1 302 Found
           as mandated in <xref target="fetching-es"/>.
 	</t>
       </section>
+
+      <section title="Federation Discovery and Trust Chain Resolution Patterns" anchor="discovery_patterns">
+	<t>
+	  This section describes different patterns that implementers may use for discovering
+	  entities within a federation and for resolving Trust Chains. It is important to distinguish
+	  between two related but distinct concepts:
+	  <list style="symbols">
+	    <t>
+	      <strong>Discovery</strong>: The process of finding entities that are part of a federation,
+	      typically to build directories or catalogs of available service providers.
+	    </t>
+	    <t>
+	      <strong>Trust Chain Resolution</strong>: The process of building and validating a Trust Chain
+	      from a known entity to a Trust Anchor, as described in <xref target="resolving_trust"/>.
+	      This process is also known as Federation Entity Discovery in this specification (see the
+	      definition in the Terminology section).
+	    </t>
+	  </list>
+	</t>
+
+	<t>
+	  While these patterns may involve both discovery and Trust Chain resolution, they serve different
+	  purposes and may be used independently depending on the use case. For example, a discovery service
+	  building a directory of OpenID Providers may collect entity identifiers without necessarily
+	  resolving Trust Chains for all discovered entities, since the actual Trust Chain resolution
+	  may occur later on, for instance: when an RP and OP engage in an authentication transaction,
+    and according to the Trust Chain resolution process described in <xref target="resolving_trust"/>.
+	</t>
+
+	<t>
+	  Implementers may support one or more of the following patterns:
+	  <list style="symbols">
+	    <t>
+	      <xref target="top_down_discovery">Top-Down Discovery</xref>: Finding entities within a federation
+	      by starting from a Trust Anchor and traversing down the hierarchy.
+	    </t>
+	    <t>
+	      <xref target="bottom_up_discovery">Bottom-Up Trust Chain Resolution</xref>: Resolving Trust Chains
```

Comment:
I suggest reordering this list and the following subsections to place bottom up discovery first in the descriptions, as that is the primary discovery mechanism defined by the specification.  We should lead with it.  Thanks.